app/views/subjects/_subject_with_tailwind.html.erb

```diff
@@ -0,0 +1,12 @@
+<li class="flex gap-x-4 px-3 py-5 justify-between items-center">
+  <div class="min-w-0">
+    <p class="text-sm/6 text-gray-900"><%= display_name(subject) %></p>
```

Comment:
Worth noting that now we are not cutting the name of the subject if it's too large:

<img width="374" alt="image" src="https://github.com/user-attachments/assets/cd49ecc6-0311-43c1-990d-a98374c21d8c" />

This was the default for the [Narrow with sticky headings](https://tailwindcss.com/plus/ui-blocks/application-ui/lists/stacked-lists#component-cb948bdbe04c717def4a7195a285d08b) list component from TailwindUI.

Previously, it looked like this:

<img width="376" alt="image" src="https://github.com/user-attachments/assets/0374a81b-1bc2-46fa-b68c-b3fbec5d2caf" />

I personally like it more with the text wrapped as we can see the full name of the subject.